ice/classes/sanger_analysis.py

```diff
@@ -391,14 +392,18 @@ def _calculate_inference_window(self):
         # set inference_window
         self.inference_window = (left_offset, iw_right_boundary)
 
+    def _should_skip_proposal(self, indel_size):
+        """
+        Indels larger than min cutoff that overlap in size with donor are not considered as an edit proposal
+        """
+        return self.donor_odn and self.HDR_OVERLAP_FILTER_CUTOFF < indel_size == self.donor_alignment.hdr_indel_size
```

Comment:
This line is not clear.  There are too many logical operations in the right hand side.  Can you clean it up?